packages/jsts/tests/analysis/analyzer.test.ts

```diff
@@ -133,14 +133,11 @@ describe('analyzeJSTS', () => {
     const tsConfigs = [path.join(__dirname, 'fixtures', 'vue_ts', 'tsconfig.json')];
     const language = 'ts';
 
-    const {
-      issues: [issue1],
-    } = analyzeJSTS(await jsTsInput({ filePath, tsConfigs }), language) as JsTsAnalysisOutput;
-    expect(issue1).toEqual(
-      expect.objectContaining({
-        ruleId: 'strings-comparison',
-      }),
-    );
+    const { issues } = analyzeJSTS(
+      await jsTsInput({ filePath, tsConfigs }),
+      language,
+    ) as JsTsAnalysisOutput;
```

Comment:
is the "as JsTsAnalysisOutput" necessary? Isn't that the exact return of the method?
```suggestion
    );
```